### PR TITLE
Content - superblock clarify

### PR DESCRIPTION
**Pull request by thephez**

### CODE REVIEW

File: _includes/devdoc/guide_dash_features.md [hunk 1]
```diff
@@ -730,6 +730,24 @@ voted to delete them.
 Sentinel manages superblock creation, voting, and submission to dashd for
 network propagation.
 
+Beginning ~3 days (1662) blocks prior to a superblock, Sentinel selects one
```
Comment (nmarley) [inline _includes/devdoc/guide_dash_features.md]: > 3 days (1662) blocks

Maybe move the word "blocks" inside the parens?

File: _includes/devdoc/guide_dash_features.md [hunk 2]
```diff
@@ -730,6 +730,24 @@ voted to delete them.
...
+masternode per block to rank proposals. This ranking is used to determine
+what a candidate superblock (or "superblock trigger") should contain. Based on
+the results, it either:
+
+1. Votes for an existing superblock trigger that matches the one calculated, or
+2. Creates and broadcasts a new superblock trigger if a matching one was not found
```
Comment (nmarley) [inline _includes/devdoc/guide_dash_features.md]:
```suggestion
2. Creates a new superblock trigger if a matching one was not found
```

AFAIK it only tries to broadcast if it's the winner.

https://github.com/dashpay/sentinel/blob/255f7cef7a1ad156edd66c287ca2cd039202de0f/bin/sentinel.py#L97-L100

**Comment by nmarley** [comment]
It only tries to broadcast if it wins the local election per block-height, but maybe that's too much detail or not relevant, in which case probably ok as-is.

File: _includes/devdoc/guide_dash_features.md [hunk 3]
```diff
@@ -730,6 +730,24 @@ voted to delete them.
 Sentinel manages superblock creation, voting, and submission to dashd for
 network propagation.
 
+Beginning ~3 days (1662) blocks prior to a superblock, Sentinel selects one
+masternode per block to rank proposals. This ranking is used to determine
+what a candidate superblock (or "superblock trigger") should contain. Based on
+the results, it either:
```
Comment (UdjinM6) [inline _includes/devdoc/guide_dash_features.md]:
```suggestion
the results, it creates and broadcasts a new superblock trigger if a matching one was not found.
```

File: _includes/devdoc/guide_dash_features.md [hunk 4]
```diff
@@ -730,6 +730,24 @@ voted to delete them.
...
+
+1. Votes for an existing superblock trigger that matches the one calculated, or
```
Comment (UdjinM6) [inline _includes/devdoc/guide_dash_features.md]:
```suggestion
```

File: _includes/devdoc/guide_dash_features.md (hunk 2 above)
Comment (UdjinM6) [inline _includes/devdoc/guide_dash_features.md]:
```suggestion
```

File: _includes/devdoc/guide_dash_features.md [hunk 5]
```diff
@@ -730,6 +730,24 @@ voted to delete them.
...
+2. Creates and broadcasts a new superblock trigger if a matching one was not found
+
```
Comment (UdjinM6) [inline _includes/devdoc/guide_dash_features.md]:
```suggestion
All masternodes vote for existing superblock triggers. 
```

File: _includes/devdoc/guide_dash_features.md (hunk 1 above)
Comment (thephez) [reply _includes/devdoc/guide_dash_features.md]: Oops, yep that's where it was supposed to be :-)

File: _includes/devdoc/guide_dash_features.md (hunk 2 above)
Comment (thephez) [reply _includes/devdoc/guide_dash_features.md]: @nmarley Ah, yes. Any MN can vote, but only the winner can broadcast a new one?

Comment (thephez) [reply _includes/devdoc/guide_dash_features.md]: Yes, udjin's changes made it clearer than what I had here

**Comment by thephez** [comment]
Thanks for the input!

**Pull request merged**